articles/unsafe.html

```diff
@@ -109,7 +109,7 @@ <h3>About the <code>unsafe</code> Standard Package</h3>
 <p>
 Although the return results of calls of any of the three functions
 are consistent in the same program,
-they might be different crossing operation systems of different architectures,
+they might be different crossing operating systems of different architectures,

```

Comment:
Again, my fault, it should be "crossing different architectures" for preciseness.

Reply:
Maybe, "are operation system and architecture dependent" should be more precise.

Reply:
I have changed this place to the following locally.

they might be different crossing operating systems, crossing architectures,
crossing compilers, and crossing compiler versions.